worker/simple.go

```diff
@@ -5,8 +5,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?